src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar_container">
```

Comment:
Consider using a semantic tag such as `<header>` or `<section>` for the search bar container instead of a non-semantic `<div>`, to improve the structure and SEO of your page.